### DISCUSSION TITLE
Questions about the treatment of diagnostic ice formation?

### DISCUSSION THREAD

**Discussion by fridlind** [ARM-Development/comble-mip#38] [Q&A] [answered]
In the main setup description for diagnostic ice formation, it says Ni = 25/L as "minimum total". What does "minimum total" mean exactly? 

It also states that homogeneous freezing should be the only other ice formation mechanism. So does this mean that the diagnostic ice formation is supposed to represent heterogeneous ice formation? But that is linked to drop loss by freezing, so it's not totally clear how this should be implemented.

**Comment by fridlind** [comment, marked as answer]
By "minimum total", the intention is that if the "qc+qr>1e-6 kg/kg and T<268.15 K" condition is met, and total Ni is below 25/L, then nucleate sufficient additional ice to reach 25/L. Note: values of Ni>25/L could result from homogeneous freezing, SGS turbulent mixing, etc.

Yes, diagnostic ice formation is intended to represent heterogeneous ice formation in the immersion mode, similar to past Arctic stratus MIPs ([Morrison et al., 2011](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2011MS000066); [Ovchinnikov et al., 2014](https://agupubs.onlinelibrary.wiley.com/doi/10.1002/2013MS000282)). It is anticipated that models would represent this by moving droplets to an ice class. In the case that insufficient droplets were to exist (perhaps below liquid cloud base), it is anticipated that the minimum total would not be attainable locally for that reason.

Clarification has been added to the [Main Model Configuration](https://arm-development.github.io/comble-mip/main_configuration.html) (see Notes).

**Reply by jchylik** [reply]
> **Ni is below 25/L, then nucleate sufficient additional ice to reach 25/L**

Thank you for the explanation, it is starting to look more feasible now. 

However, there is still an issue of the  growth of hydrometeors.
[Ovchinnikov et al., 2014](https://doi.org/10.1002/2013MS000282 "Ovchinnikov et al., 2014") prescibes:

> _Once formed, ice crystals are subjected to diffusional growth and sublimation, gravitational settling, resolved advection, and subgrid-scale mixing. In the simulations analyzed here, ice particles grow only through water vapor deposition. (Though average ice crystal size can change from sedimentation-induced size sorting.) Riming and aggregation are turned off._ 

Should we also follow that?

**Reply by fridlind** [reply]
The ice crystal properties and processes in the Ovchinnikov et al. ISDAC case were deliberately idealized and highly specified (despite uncertainties) in order to limit as many of the sources of differences between the participating models as possible. Thus, aggregates were neglected even though aggregation was an active process in that case, as in the similar ISDAC case simulated by Avramov et al. ([2011](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2011JD015910)). Thermodynamic conditions were also nudged, which limited the degree to which feedbacks could be expressed between microphysics and thermodynamics.

Here we are focused more on comparison of models with observations as a benchmark for climate model development in SCM mode. We therefore refer to this MIP as a Model-observation Intercomparison Project. Unlike the ISDAC MIP, we are also including climate models in SCM mode, where nudging is avoided in order to test the impact of microphysics choices on thermodynamic evolution. An additional contrast to the ISDAC cases, which showed many pristine vapor-grown crystals and their aggregates but no active riming, is that riming is likely important owing to much higher liquid water path and colder cloud top temperates. Unfortunately we have no in situ measurements to confirm ice morphological signatures of riming, vapor growth and aggregation. But it is probably a safe bet that particles during much of the Lagrangian path are similar in morphology to the M-PACE MIP case, which was less supercooled but reached similarly high liquid water path. In the M-PACE case, many highly rimed particles were present, mixed with relatively pristine particles and a variety of novel habits resulting from cycles of vapor growth and riming, as discussed by Fridlind et al. ([2007](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2007JD008646)). A contrast of ISDAC and M-PACE CPI images is made in Fridlind and Ackerman ([2018](https://www.sciencedirect.com/science/article/abs/pii/B9780128105498000076?via%3Dihub)); Figure 7 from that chapter shows a small sample of M-PACE CPI imagery and is reproduced below.

In summary, in this case we are specifying only the ice crystal formation diagnostically, similar to the earlier Morrison et al. SHEBA case. Thank you for the question, and I hope that we are able to use forward simulation of Doppler velocities at the Andenes site to evaluate submitted simulations. For that purpose, we have requested 3D mass-weighted fall speeds from LES models as listed [here](https://arm-development.github.io/comble-mip/notebooks/output/output_list.html). An example of observed and forward-simulated Doppler velocity fields is shown [here](https://arm-development.github.io/comble-mip/emc2.html) (2nd panel from the top), based on a preliminary DHARMA simulation. These fields reflect the combined effect of dynamics and reflectivity-weighted hydrometeor fall speeds, thus yielding signatures of the latter that can be probed for realism of ice properties in the LES simulations, as shown also in Avramov et al. for an ISDAC case.

![Screenshot 2024-01-26 at 1 35 33 PM](https://github.com/ARM-Development/comble-mip/assets/20172705/36e46121-1d21-4116-a711-c0f2fa7e80af)

**Reply by jchylik** [reply]
Thank you for the explanatory answer!

> 3D mass-weighted fall speeds from LES models 

Of course. That said, shall we use a default geometry in each model, or the geometry  from [Ovchinnikov et al., 2014](https://doi.org/10.1002/2013MS000282) ?

>> _The initial size of newly nucleated ice crystals is set to D =10 µm in bulk models or specified as the smallest allowable size in the bin models.
>> The ice properties formulated above represent an idealization of dendrites as spheres of constant and low equivalent density._

**Reply by fridlind** [reply]
Please use your default model geometry. Also, if you encounter sensitivity to model choices, please feel free to submit a sensitivity test with the naming convention described [here](https://arm-development.github.io/comble-mip/timeline.html).

Many thanks,
Ann
